src/headers.ts

```diff
@@ -2,5 +2,6 @@ export { ContentDispositionInit, ContentDisposition } from './lib/content-dispos
 export { ContentTypeInit, ContentType } from './lib/content-type.js';
 export { CookieInit, Cookie } from './lib/cookie.js';
 export { SetCookieInit, SetCookie } from './lib/set-cookie.js';
+export { CacheControlInit, CacheControl } from './lib/cache-control.js';
```

Comment:
Put this one at the top of the file since Cache-Control comes before Content-Disposition alphabetically.